connector/linkedin/linkedin.go

```diff
@@ -149,15 +157,46 @@ func (p profile) fullname() string {
 	return fname
 }
 
+func (c *linkedInConnector) primaryEmail(ctx context.Context, client *http.Client) (email string, err error) {
```

Comment:
I'd prefer non-named return values, but that's mostly a matter of taste and not consistent even within this code base. So, let's leave it as is, since we're not using _naked returns_, there's little harm done. 😉

Reply:
Okay.